<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BorcVerDialog</class>
 <widget class="QDialog" name="BorcVerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Borç Ver</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="2">
    <widget class="QLineEdit" name="input_miktar">
     <property name="placeholderText">
      <string>Miktar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Borç Alınma Tarihi</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="borc_ver_text">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDateEdit" name="borc_tarihi">
     <property name="maximumDateTime">
      <datetime>
       <hour>23</hour>
       <minute>59</minute>
       <second>59</second>
       <year>2050</year>
       <month>12</month>
       <day>31</day>
      </datetime>
     </property>
     <property name="minimumDateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2017</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="date">
      <date>
       <year>2019</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDateEdit" name="vade_tarihi">
     <property name="maximumDateTime">
      <datetime>
       <hour>23</hour>
       <minute>59</minute>
       <second>59</second>
       <year>2050</year>
       <month>12</month>
       <day>31</day>
      </datetime>
     </property>
     <property name="minimumDateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2018</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="date">
      <date>
       <year>2019</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Borç Ödeme Tarihi</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="btn_borc_ver">
     <property name="text">
      <string>Borç Ver</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
